src/common/util/src/file_util.cpp

```diff
@@ -351,14 +352,26 @@ std::wstring ov::util::string_to_wstring(const std::string& string) {
 std::string ov::util::get_absolute_file_path(const std::string& path) {
     std::string absolutePath;
     absolutePath.resize(MAX_ABS_PATH);
-    auto absPath = get_absolute_path(&absolutePath[0], path);
-    if (!absPath) {
-        std::stringstream ss;
-        ss << "Can't get absolute file path for [" << path << "], err = " << strerror(errno);
-        throw std::runtime_error(ss.str());
+    (void*)get_absolute_path(&absolutePath[0], path);
```

Comment:
The original problem is that Linux `realpath` checks file existence and read access, but `_fullpath` on Windows - no. And to guarantee full path generation without exceptions I need to skip/workaround that check. It's required to prevent such exceptions on plugins registration step in order to postpone failure on plugin loading step (e.g. if there is no GNA plugin file, I expect failure only on loading of GNA, but not on registration of paths to CPU/GPU/GNA etc. common step)

According to https://man7.org/linux/man-pages/man3/realpath.3.html, `realpath() returns a pointer to the resolved_path` (i.e. `&absolutePath[0]` and return pointer will be same). But in notes:
>    GNU extensions
       If the call fails with either EACCES or ENOENT and resolved_path
       is not NULL, then the prefix of path that is not readable or does
       not exist is returned in resolved_path.

So, if file doesn't exist or no access, `realpath` will return NULL, `&absolutePath[0]` is undefined, but according to notes it should contain the path that we expect. So with the change I just make a workaround on `EACCES or ENOENT` errors

To be sure it works, I checked it with GCC and CLANG, and both works. Otherwise I expect `&absolutePath[0]` should contain NULL as `realpath` return value (but it wasn't guaranteed). 

P.S. Windows `_fullpath` is very close to Linux `realpath`, but doesn' check existence and access (it checks correctness of disk letter or size of char array, but I can't achieve NULL from `_fullpath`). Details: https://learn.microsoft.com/ru-RU/cpp/c-runtime-library/reference/fullpath-wfullpath?view=msvc-170#return-value

Reply:
I may add more strict checks like:
`
retval = get_absolute_path(&absolutePath[0], path);
if (retval == NULL)
    if (errno == ENOENT || errno == EACCES)  // error codes are equal both on Win or Lin
        if (&absolutePath[0])
            return &absolutePath[0];
    throw std::runtime_error;
return retval;

`